hare3/protocol.go

```diff
@@ -482,7 +482,9 @@ func thresholdTallies[T interface {
 	comparable
 	fmt.Stringer
 }](
-	state map[messageKey]*gossipInput, filter IterRound, msgGrade grade,
+	state map[messageKey]*gossipInput,
+	filter IterRound,
+	msgGrade grade,
```

Comment:
why did you change this? i think it passes linter

i don't mind that the formatting change, but it makes it harder to focus on code of the change

Reply:
I think I was going through the hare code to understand it and I reformatted it to be able to read it and then staged the change brainlessly.